problem_builder/public/css/questionnaire.css

```diff
@@ -78,7 +82,12 @@
 }
 
 .mentoring .choices-list .choice-selector {
-    display: inline-block;
+    display: table-cell;
+}
+
+.mentoring .choices-list .choice-label-text {
+    display: table-cell;
+    padding-left: 0.4em;
```

Comment:
We should add some padding to the right as well - currently there's no padding between the choice text and the line that divides the text from the tip:

![screen shot 2017-01-27 at 09 51 01](https://cloud.githubusercontent.com/assets/32585/22365306/01583dfc-e477-11e6-86a3-2293b4aa2a5b.png)

Ideally it would be the same amount of padding on both sides, to make it easier to support RTL.